install_config/install/host_preparation.adoc

```diff
@@ -336,7 +336,8 @@ file system is located.
 
 Option A is the most robust option, however it requires adding an additional
 block device to your host before configuring Docker storage. Options B and C
-both require leaving free space available when provisioning your host.
+both require leaving free space available when provisioning your host. The
```

Comment:
how about something more direct?
Option C is known to cause issues with some applications, for example RHMAP.